src/lib/annotations/doc/DocAnnotator.js

```diff
@@ -499,25 +500,6 @@ class DocAnnotator extends Annotator {
         });
     }
 
-    /**
-     * Checks whether mouse is inside any dialog on the current page
-     *
-     * @protected
-     * @param {Event} event - Mouse event
-     * @param {number} page - Current page number
-     * @return {boolean} Whether or not mouse is inside a dialog on the page
-     */
-    isInDialogOnPage(event, page) {
```

Comment:
We used to use this method in the monstrous highlightMouseMove() method but not that we're simply checking the datatype being hovered over, we no longer need this!

Reply:
👍